release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3417,4 +3417,23 @@ $ oc adm release info 4.10.32 --pullspecs
 [id="ocp-4-10-32-upgrading"]
 ==== Updating
 
-To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-33"]
+=== RHBA-2022:6532 - {product-title} 4.10.33 bug fix and security update
+
+Issued: 2022-09-20
+
+{product-title} release 4.10.33, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:6532[RHBA-2022:6532] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:6531[RHSA-2022:6531] advisory.
```

Comment:
These links seem to give a 404 error for me?
Otherwise looks good!

Reply:
The links won't work until it ships.

Reply:
ACK, thanks!